examples/pytorch/speech-pretraining/run_wav2vec2_pretraining_no_trainer.py

```diff
@@ -368,7 +368,7 @@ def main():
     send_example_telemetry("run_wav2vec2_pretraining_no_trainer", args)
 
     # Initialize the accelerator. We will let the accelerator handle device placement for us in this example.
-    accelerator = Accelerator()
+    accelerator = Accelerator(gradient_accumulation_steps=args.gradient_accumulation_steps)
```

Comment:
This one does a lot more things in its training loop, so would leave it as it was as I'm more afraid to break anything.

Reply:
Removed changes